src/flex-example-app-android.xml

```diff
@@ -72,6 +76,14 @@
                     <activity android:name="com.vungle.publisher.FullScreenAdActivity" android:configChanges="keyboardHidden|orientation|screenSize" android:theme="@android:style/Theme.NoTitleBar.Fullscreen"    />
                     <service android:name="com.vungle.publisher.VungleService" android:exported="false"/>
 
+
+<activity android:name="com.inmobi.rendering.InMobiAdActivity" android:configChanges="keyboardHidden|orientation|keyboard|smallestScreenSize|screenSize" android:theme="@android:style/Theme.Translucent.NoTitleBar" android:hardwareAccelerated="true" /><receiver android:name="com.inmobi.commons.core.utilities.uid.ImIdShareBroadCastReceiver" android:enabled="true" android:exported="true" >
+  <intent-filter>
+    <action android:name="com.inmobi.share.id" />
+  </intent-filter>
+</receiver>
+                    <activity android:name="com.inmobi.androidsdk.IMBrowserActivity" android:configChanges="keyboardHidden|orientation|keyboard|smallestScreenSize|screenSize" android:theme="@android:style/Theme.Translucent.NoTitleBar" android:hardwareAccelerated="true" />
```

Comment:
[their docs](https://support.inmobi.com/monetize/integration/partner-platforms/adobe-air-partner-platform-sdk-integration-guide/#configuration) don't say this second activity is required - was this something that came up when you emailed them?